crates/nu-utils/standard_library/dirs.nu

```diff
@@ -0,0 +1,80 @@
+# Maintain a list of working directories and navigates them
```

Comment:
maybe remove the "s" from "navigate" for consistency with "Maintain"?

Reply:
Should be "maintains ... and navigates ..."